images/regen-sandbox/bootstrap-solo-network.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
```

Comment:
Updating this might resolve @wgwz's issue.

```suggestion
#!/usr/bin/env bash
```